src/main/java/org/mockito/internal/stubbing/defaultanswers/ForwardsInvocations.java

```diff
@@ -40,6 +40,7 @@ public Object answer(InvocationOnMock invocation) throws Throwable {
             }
 
             Object[] rawArguments = ((Invocation) invocation).getRawArguments();
+            delegateMethod.setAccessible(true);
```

Comment:
We should probably `try-catch` this one, in case we are unable to set it accessible. However, in that case I think we should still continue with calling `delegateMethod.invoke`. E.g. the try-catch-clause should be only around this statement, but not the other statements.